google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -187,6 +193,26 @@ public static BlobSourceOption decryptionKey(String key) {
   }
 
   /**
+   * Downloads this blob to the given file path.
+   *
+   * @param path destination
+   * @throws IOException upon failure
+   */
+  public void downloadTo(Path path) throws IOException {
```

Comment:
I believe the convention is to throw StorageException upon failure. Also it is considered a good practice to avoid throwing of checked exceptions when applicable (there is no universal agreement, but checked exceptions have bad reputation these days, for example http://www.artima.com/intv/handcuffs.html).